lib/pattern-preset.js

```diff
@@ -77,6 +77,11 @@ export const presets = {
     ],
   },
 
+  RequirementsTxt: {
+    pathSubstrings: ['requirements.txt'],
+    githubClasses: [],
```

Comment:
Github uses the `type-text` class for requirements.txt files, which seems too general to use for filetype detection.